generative/maisi/large_files.yml

```diff
@@ -0,0 +1,23 @@
+large_files:
```

Comment:
According to the conversation, the large files Google Drive links will be removed/replaced by links to NGC/MONAI extra data

Reply:
I don't think we want to maintain this large_files here, right? Just download them where they need such as in the inference notebook.

Reply:
Thank you! It seems we discussed that we will resolve it in the next PR after then new link is ready

Reply:
large_files.yml removed. Thanks